src/components/Outputs.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const Outputs = ({ amount, result }) => {
+
+  const calculateTotal = (amount, result) => {
+    const total = Number(amount) + result;
+    return formatMoney(total)
+  }
+
+  const formatMoney = money => {
```

Comment:
Its best practice to write functions before you call them, and it looks like it is being called above at line 7.